skycoin/dmsg
# Fix dmsg server

Fixes #	

 Changes:	
- Changed `httpserver` of `dmsg-server` to run on port `8082` instead of `8081`
- Added readme for `dmsg-server`
- Updated the URL of prod `dmsg-discovery`
- Ran `go mod vendor`

How to test this PR:
1. Run the changes in integration env in `skywire-services`
2. Check `localhost:8082/health`
3. Check if the visors connect to the dmsg network (try multiple times)

The image is also running on myownsky test env and can be tested via `http://dmsg-discovery.myownsky.xyz`.
The health endpoint of the `dmsg-server` is available on `http://165.232.178.203:30153/health`
